src/go/k8s/apis/vectorized/v1alpha1/console_types.go

```diff
@@ -221,6 +221,32 @@ type ConsoleStatus struct {
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
 
 	Connectivity *Connectivity `json:"connectivity,omitempty"`
+
```

Comment:
Because previous assertion was wrong and whole test silently pass, but console deployment was never created. 

https://github.com/redpanda-data/redpanda/blob/2434e021f8685ffce55c99eba7bc3ef4dff790be/src/go/k8s/tests/e2e/superusers-prefix/02-assert.yaml#L1-L16

vs

https://github.com/joejulian/redpanda/blob/702478923635b919004c6ed1b105fd7ca6c10f83/src/go/k8s/tests/e2e/superusers-prefix/01-assert.yaml